JSDemos/Demos/LoadPanel/Overview/description.md

```diff
@@ -1 +1,16 @@
-The LoadPanel is&nbsp;an&nbsp;overlay UI component notifying the viewer that loading is&nbsp;in&nbsp;progress.
\ No newline at end of file
+The LoadPanel is an overlay component used to notify users that a process is in progress. In this demo, you can see how to initialize and configure the component.
+
+## Apply Indicator and Pane
+
+LoadPanel allows you to display a load indicator and a pane behind the indicator. For this, enable the [showIndicator](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#showIndicator) and [showPane](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#showPane) properties. Uncheck the *With indicator* and *With pane* checkboxes to disable the indicator and pane.
```

Comment:
- These properties are already enabled by default. Please rewrite this paragraph.
- There's also a text string displayed by the LoadPanel. You can specify it using the `message` property.